core/src/database/migrations.cpp

```diff
@@ -1205,5 +1198,29 @@ namespace ledger {
             sql << "DROP INDEX bitcoin_operations_transaction_uid_index ;";
         }
 
+        template <> void migrate<29>(soci::session& sql, api::DatabaseBackendType type) {
+            sql << "ALTER TABLE bech32_parameters ADD p2trversion VARCHAR(255) DEFAULT 0x01;";
+        }
+
+        template <> void rollback<29>(soci::session& sql, api::DatabaseBackendType type) {
+            // SQLite doesn't handle ALTER TABLE DROP
+            if (type != api::DatabaseBackendType::SQLITE3) {
+                sql << "ALTER TABLE bech32_parameters DROP p2trversion";
+            } else {
+                sql << "CREATE TABLE bech32_parameters_swap("
```

Comment:
I understand what you have to do here. For maintainance purpose, can you just use the same "string" for the `CREATE TABLE bech32_parameters*` queries (the other one on line 439)? It could be a `std::string getBech32ParamsCreateTableSQL(const char* tableName);`.

Reply:
done